kotest-assertions/kotest-assertions-json/src/jvmMain/kotlin/io/kotest/assertions/json/JsonTypes.kt

```diff
@@ -0,0 +1,18 @@
+package io.kotest.assertions.json
+
+/**
+ * JSON element representation via Kotlin String.
+ *
+ * It can be any JSON value, for example:
+ * * JSON string: `""id""` – please note that it differs from a simple Kotlin string `"id"` which is not valid JSON representation.
+ * * JSON number: `"42"`.
+ * * JSON null: `"null"`.
+ * * JSON object: `"{"id": 42}"`.
+ * * JSON array: `"["id", 42, null]"`.
+ */
+typealias Json = String
+
+/**
+ * String representing a key for JSON value. For example, to access `42` in `"{"id": 42}"`, use `"id"` as a key.
+ */
+typealias JsonKey = String
```

Comment:
Would the user need to import these typealias?
I don't see anything wrong in having them in Kotest, but propagating them users' code seems a little bit odd to me

Reply:
If a user just calls functions which have these aliases as receivers or parameters, there is no need to import. If `Json` is inferred when assigned to a variable, no imports are needed too.

If a user wants to explicitly declare a symbol with `Json` type, then they should import it. However, using `String` instead of `Json` doesn't require imports and is allowed by the compiler without any warnings.